src/Makefile.am

```diff
@@ -926,8 +923,7 @@ tests_testURL_LDFLAGS = $(LIBADD_DL)
 
 check_PROGRAMS += tests/testDns
 tests_testDns_SOURCES = \
-	tests/testRFC1035.cc \
-	tests/testRFC1035.h
+	tests/testRFC1035.cc
```

Comment:
Should we fix capitalization of the affected files in _this_ PR? For example, testRFC1035.cc should (at some point), become TestRfc1035.cc (to match the primary TestRfc1035 class in that file). Should that happen in this PR? I am just asking a question, not implying a "yes" answer :-).

Reply:
> Should we fix capitalization of the affected files in _this_ PR?

That feels like scope creep, I prefer to focus on one thing at a time, so no. A followup PR can take care of that.